stable_baselines3/common/policies.py

```diff
@@ -639,17 +687,21 @@ def get_distribution(self, obs: th.Tensor) -> Distribution:
         :return: the action distribution.
         """
         features = self.extract_features(obs)
+        if not self.share_features_extractor:
```

Comment:
I think it would be cleaner to add a `extract_actor_features()` and `extract_critic_features()` methods

Reply:
I agree, I'll do it!